test/unit/core/type_traits/range_iterator_test.cpp

```diff
@@ -233,23 +233,23 @@ TEST(range_and_iterator, dimension)
     EXPECT_EQ(2u, seqan3::range_dimension_v<std::ranges::iterator_t<std::vector<std::vector<int>>>>);
 }
 
-TEST(range_and_iterator, compatible)
+TEST(range_and_iterator, range_compatible)
 {
-    EXPECT_TRUE((seqan3::compatible<std::vector<int>,
+    EXPECT_TRUE((seqan3::range_compatible<std::vector<int>,
                                     std::list<int>>));
```

Comment:
can you re-align them?

```suggestion
                                         std::list<int>>));
```